src/js/colors/colors.test.ts

```diff
@@ -1,452 +0,0 @@
-    expect(importableColors.dark.error.muted).toStrictEqual(
-      designTokens.dark.colors.error.muted.value,
-    );
-  });
-
-  it('js tokens for error.inverse matches figma tokens error.inverse', () => {
-    expect(importableColors.dark.error.inverse).toStrictEqual(
-      designTokens.dark.colors.error.inverse.value,
-    );
-  });
-
-  it('js tokens for warning.default matches figma tokens warning.default', () => {
-    expect(importableColors.dark.warning.default).toStrictEqual(
-      designTokens.dark.colors.warning.default.value,
-    );
-  });
-
-  it('js tokens for warning.alternative matches figma tokens warning.alternative', () => {
-    expect(importableColors.dark.warning.alternative).toStrictEqual(
-      designTokens.dark.colors.warning.alternative.value,
-    );
-  });
-
-  it('js tokens for warning.muted matches figma tokens warning.muted', () => {
-    expect(importableColors.dark.warning.muted).toStrictEqual(
-      designTokens.dark.colors.warning.muted.value,
-    );
-  });
-
-  it('js tokens for warning.inverse matches figma tokens warning.inverse', () => {
-    expect(importableColors.dark.warning.inverse).toStrictEqual(
-      designTokens.dark.colors.warning.inverse.value,
-    );
-  });
-
-  it('js tokens for success.default matches figma tokens success.default', () => {
-    expect(importableColors.dark.success.default).toStrictEqual(
-      designTokens.dark.colors.success.default.value,
-    );
-  });
-
-  it('js tokens for success.alternative matches figma tokens success.alternative', () => {
-    expect(importableColors.dark.success.alternative).toStrictEqual(
-      designTokens.dark.colors.success.alternative.value,
-    );
-  });
-
-  it('js tokens for success.muted matches figma tokens success.muted', () => {
-    expect(importableColors.dark.success.muted).toStrictEqual(
-      designTokens.dark.colors.success.muted.value,
-    );
-  });
-
-  it('js tokens for success.inverse matches figma tokens success.inverse', () => {
-    expect(importableColors.dark.success.inverse).toStrictEqual(
-      designTokens.dark.colors.success.inverse.value,
-    );
-  });
-
-  it('js tokens for info.default matches figma tokens info.default', () => {
-    expect(importableColors.dark.info.default).toStrictEqual(
-      designTokens.dark.colors.info.default.value,
-    );
-  });
-
-  it('js tokens for info.alternative matches figma tokens info.alternative', () => {
-    expect(importableColors.dark.info.alternative).toStrictEqual(
-      designTokens.dark.colors.info.alternative.value,
-    );
-  });
-
-  it('js tokens for info.muted matches figma tokens info.muted', () => {
-    expect(importableColors.dark.info.muted).toStrictEqual(
-      designTokens.dark.colors.info.muted.value,
-    );
-  });
-
-  it('js tokens for info.inverse matches figma tokens info.inverse', () => {
-    expect(importableColors.dark.info.inverse).toStrictEqual(
-      designTokens.dark.colors.info.inverse.value,
-    );
-  });
-});
```

Comment:
Removing this file so we don't have to keep 2 versions of the unit tests for colors and ignoring from jest config